lib/utils.pm

```diff
@@ -1819,8 +1825,14 @@ sub install_patterns {
         }
         # if pattern is common-criteria and PATTERNS is all, skip, poo#73645
         next if (($pt =~ /common-criteria/) && check_var('PATTERNS', 'all'));
-        zypper_call("in -t pattern $pt", timeout => 1800);
-
+        if (($pt =~ /CFEngine/) && is_sle('>=12') && is_sle('<15')) {
```

Comment:
maybe you need detect error for CFEngine, then you can start workround , like patch_sle?

Reply:
Ok, updated code. Please review it again. Updated verification log also.